brainglobe_atlasapi/atlas_generation/atlas_scripts/kim_mouse.py

```diff
@@ -237,9 +236,21 @@ def create_atlas(working_dir, resolution):
 
 
 if __name__ == "__main__":
-    resolution = 10  # some resolution, in microns (10, 25, 50, 100)
+    # Create argument parser to pass resoulution as an argument
+    parser = argparse.ArgumentParser(
+        description="Create an atlas with a specified resolution."
+    )
+    parser.add_argument(
+        "--resolution",
+        type=int,
+        default=10,
+        help="Resolution in microns (10, 25, 50, 100)",
+    )
+    args = parser.parse_args()
```

Comment:
Why are we adding an argument parser here?
I think it should be a loop over the resolutions we'd like to provide?

Reply:
Yeah I think the atlas generation scripts are meant to (?) loop over resolutions.

Reply:
It didn't do it. Only one atlas was generated when I submitted a job on the HPC and I didn't want to change the resolution value in the code manually, push the new version and submit a new job for each resolution. I might be missing something.

Reply:
> I didn't want to change the resolution value in the code manually, push the new version and submit a new job for each resolution.
 
 I agree this is a worthy goal - my point is that I think this should not be achieved by a command line argument, but rather with a for loop in the code.

Reply:
I thought this was already in the code, but it's what the main script (see #273) coordinates.

Reply:
I see what you mean now. Will I still have the option to easily specify which atlas I want to generate? For example, if I only need the 100um one, I wouldn't want to wait for the script to generate all the others.

Reply:
I don't think you will, no.
If you'd like that flexibility, you probably want to clone the repository on the HPC, and edit the script via the command line?

Reply:
Having thought about this, I think it's best to:
- Stick with what you have now
- Later on, remove it and sort out the main script (unless anyone has the capacity to address #273 now?

@alessandrofelder?

Reply:
I think we commented at the same time @alessandrofelder. I think we ideally shouldn't add the parser to the script, but getting fixed versions of the atlases out should be a priority.